### PR TITLE
update readme to link to lob docs

### PR DESCRIPTION
**Pull request by ami**
@pon @mgartner @elnaz

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,768 +102,102 @@ Lob.setApiVersion("apiVersion");
...
-// List All Services
-final ListenableFuture<ServiceResponseList> services = client.getServices();
-```
-
-## Simple Postcard Service
+## Examples
+
+We've provided various examples for you to try out [here](https://github.com/lob/lob-java/tree/master/lob-java-examples/src/main/java/com/lob/examples).
+
+There are simple scripts to demonstrate how to create all the core Lob objects (checks, letters, postcards. etc.) As well as more complex examples that utilize other libraries and external files:
```
Comment (robinjoseph08) [inline README.md]: postcards,

**Comment by robinjoseph08** [comment]
LGTM

**Comment by ami** [comment]
@robinjoseph08 updated :smile_cat:

**Comment by mgartner** [comment]
LGTM

**Pull request merged**